src/main/thrift/parquet.thrift

```diff
@@ -151,14 +151,14 @@ enum ConvertedType {
   /**
    * An embedded JSON document
    *
-   * A JSON document embedded within a single UTF8 column.
+   * A JSON document embedded within a single BYTE_ARRAY(STRING) column.
```

Comment:
Should we keep it as is? This is the deprecated `ConvertedType` section where `UTF8` is used for string type.

Reply:
Oh, good point. I'll revert.